src/widgets/AutocompleteSelector.js

```diff
@@ -95,11 +97,10 @@ export class AutocompleteSelector extends React.PureComponent {
           autoCapitalize="none"
           autoCorrect={false}
           autoFocus
-          color="white"
-          onChange={text => this.setState({ queryText: text })}
           placeholder={placeholderText}
-          placeholderTextColor="white"
+          onChangeText={this.onChangeText}
           style={[localStyles.text, localStyles.searchBar]}
+          color="white"
```

Comment:
Can we use the constant `WHITE` here?

This one defined here:
https://github.com/openmsupply/mobile/blob/78406fdd118175f491b99db6c82cb79eb7006a09/src/globalStyles/colors.js#L21

It would be good to export it in the `index` file - it seems someone forgot!!

https://github.com/openmsupply/mobile/blob/78406fdd118175f491b99db6c82cb79eb7006a09/src/globalStyles/index.js#L26-L41

Reply:
@joshxg 
@Chris-Petty 
There are 2 changes regarding to export and import WHITE = '#FFFFFF' to search bar.